test/model/mapping/coercer_test.rb

```diff
@@ -16,22 +17,41 @@
       coercer.to_record(entity).must_equal(name: 'Tyrion Lannister')
     end
 
+    describe 'new record' do
+      # Bug:
+      # https://github.com/lotus/model/issues/155
+      it 'ignores unset values' do
+        entity = User.new(name: 'Daenerys Targaryen')
+        coercer.to_record(entity).must_equal(name: 'Daenerys Targaryen')
```

Comment:
In this test case we should ensure that if a attribute is not explicitly set to nil we wont set it to nil, something like:

```ruby
entity = User.new(name: 'Daenerys Targaryen')
coercer.to_record(entity).wont_equal(name: 'Daenerys Targaryen', age: nil)
```